README.md

```diff
@@ -263,73 +182,6 @@ Note that `chain-id` should be specified as text e.g. `SN_GOERLI`.
 
 This can be used to interact with a custom StarkNet gateway, or to use a gateway proxy.
 
-## Running with Docker
-
-The `pathfinder` node can be run in the provided Docker image.
-Docker image is the easiest way which does not involve a lot of python setup.
-The following assumes you have [docker installed](https://docs.docker.com/get-docker/) and ready to go.
-
-The example uses `$HOME/pathfinder` as the volume directory where persistent files used by `pathfinder` will be stored.
-It is easiest to create the volume directory as the user who is running the docker command.
-If the directory gets created by docker upon startup, it might be unusable for creating files.
-
-```bash
-# ensure the directory has been created before invoking docker
-mkdir -p $HOME/pathfinder
-docker run \
-  --rm \
-  -p 9545:9545 \
-  --user "$(id -u):$(id -g)" \
-  -e RUST_LOG=info \
-  -e PATHFINDER_ETHEREUM_API_URL="https://goerli.infura.io/v3/<project-id>" \
-  -v $HOME/pathfinder:/usr/share/pathfinder/data \
-  eqlabs/pathfinder
-```
-
-### Updating the docker image
-
-When pathfinder detects there has been a new release, it will log a message similar to:
-
-```
-WARN New pathfinder release available! Please consider updating your node! release=0.1.8-alpha
-```
-
-You can try pulling the latest docker image to update it:
-
-```bash
-docker pull eqlabs/pathfinder
-```
-
-There is a chance of seeing the release notification before a new docker image is available for download, so just wait a minutes and then retry.
-
-### Available images
-
-Our images are updated on every `pathfinder` release. This means that the `:latest` docker image does not track our `main` branch here, but instead matches the latest `pathfinder` [release](https://github.com/eqlabs/pathfinder/releases).
-
-### Building the container image yourself
-
-Building the container image from source code is necessary only in special cases or development.
-You can build the image by running:
-
-```bash
-docker build -t pathfinder .
-```
-
-### Docker compose
-
-Create the folder `pathfinder` where your `docker-compose.yaml is
-
-```bash
-mkdir -p pathfinder
-
-# replace the value by of PATHFINDER_ETHEREUM_API_URL by the HTTP(s) URL pointing to your Ethereum node's endpoint
-cp example.pathfinder-var.env pathfinder-var.env
-
-docker-compose up -d
-```
-
-To check if it's running well use `docker-compose logs -f`.
-
 ## JSON-RPC API
```

Comment:
Btw: there are instances of `v0.2.1-rc1` which should be changed to just `v0.2.1`